privacyidea/static/components/config/views/config.resolvers.ldap.html

```diff
@@ -264,6 +264,15 @@
                         />
             </div>
         </div>
+        <div class="form-group">
+            <label for="multivalueattributes" class="col-sm-3 control-label"
+                translate>Mutlivalue Attributes</label>
```

Comment:
*Multivalue :)

Reply:
Fixed it in 6369f5c0a847a9061e4c6433cd3b359c8721ed79